gulpfile.js

```diff
@@ -23,7 +23,7 @@ var sources = {
   content: 'app/**/*.{markdown,md,html,txt,yml,yaml}',
   styles: paths.assets + 'stylesheets/**/*',
   js: [
-    paths.assets + 'javascripts/**/*.js',
```

Comment:
I don't think this should be changed. It also doesn't appear to be in the scope of this PR?